setup.py

```diff
@@ -7,14 +7,15 @@
     "yarl>=1.6,<2.0",
 ]
 
-scripts = [
-    "scripts/gql-cli",
+console_scripts = [
+    "gql-cli=gql.cli:gql_cli",
 ]
 
 tests_requires = [
     "parse==1.15.0",
     "pytest==6.2.5",
     "pytest-asyncio==0.16.0",
+    "pytest-console-scripts==1.3.1",
```

Comment:
i tried doing this directly, but there's a lot of care and feeding of thread/processes, coverage, ioloops, etc. that this takes care of.

Reply:
That's perfectly fine